src/Init/Init_ByRestart_HDF5.cpp

```diff
@@ -1440,7 +1483,7 @@ void Check_Makefile( const char *FileName, const int FormatVersion )
    Makefile_t RS;    // RS = ReStart
 
    LoadField( "Model",                  &RS.Model,                  SID, TID, NonFatal, &RT.Model,                  1,    Fatal );
-   LoadField( "Gravity",                &RS.Gravity,                SID, TID, NonFatal, &RT.Gravity,                1,    Fatal );
+   LoadField( "Gravity",                &RS.Gravity,                SID, TID, NonFatal, &RT.Gravity,                1, NonFatal );
```

Comment:
This change is fine but I want to make sure that this is indeed what you want. Which test requires it?